Source/KeyHandler.mm

```diff
@@ -1420,37 +1425,39 @@ - (BOOL)_handleCandidateState:(InputState *)state
             if (isPlusKey) {
                 InputStateCustomMenuEntry *boost = [[InputStateCustomMenuEntry alloc] initWithTitle:NSLocalizedString(@"Boost", @"")
                                                                                            callback:^{
-                    __strong __typeof(weakSelf) strongSelf = weakSelf;
-                    if (!strongSelf) {
-                        return;
-                    }
-                    [strongSelf.delegate keyHandler:strongSelf didRequestBoostScoreForPhrase:candidate.value reading:reading];
-                    [strongSelf.delegate keyHandlerDidRequestReloadLanguageModel:strongSelf];
-                    [strongSelf _walk];
-                    InputStateInputting *inputting = (InputStateInputting *)[strongSelf buildInputtingState];
-                    stateCallback(inputting);
-                }];
+                                                                                               __strong __typeof(weakSelf) strongSelf = weakSelf;
```

Comment:
Nit: are these extra spaces inserted? Same for the following lines.

Reply:
Fixed. It was caused by editor setting.